vdev/src/commands/exec.rs

```diff
@@ -0,0 +1,29 @@
+use anyhow::{bail, Result};
+use clap::Args;
+
+use crate::app;
+
+/// Execute a command within the repository
+#[derive(Args, Debug)]
+#[command()]
+pub struct Cli {
+    #[arg(trailing_var_arg = true, allow_hyphen_values = true)]
+    args: Vec<String>,
+}
+
+impl Cli {
+    pub fn exec(&self) -> Result<()> {
+        let mut command = app::construct_command(&self.args[0]);
```

Comment:
This panics if you just `vdev exec`. It would be nice if it could output a useful error message instead.

Reply:
Fixed!